ui/src/cloud/components/OrgSettings.tsx

```diff
@@ -0,0 +1,34 @@
+// Libraries
+import {PureComponent} from 'react'
+import {connect} from 'react-redux'
+
+// Constants
+import {CLOUD} from 'src/shared/constants'
+
+// Actions
+import {getOrgSettings as getOrgSettingsAction} from 'src/cloud/actions/orgsettings'
+
+interface DispatchProps {
+  getOrgSettings: typeof getOrgSettingsAction
+}
+
+class OrgSettings extends PureComponent<DispatchProps> {
+  public componentDidMount() {
+    if (CLOUD) {
```

Comment:
good defensive coding here. is this check necessary? not sure how components under the `/cloud` directory work.

edit: ah, i see lower down that this component is used in OSS